frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionStyledComponents.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button, Card, Dialog } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+
```

Comment:
Note that making this into a dict instead of defining it in the file means that we don't get a warning if a style is no longer in use

Reply:
It does make important quite compact though and is similar to how some EDS components work, eg tokens

Reply:
I think this will help us reduce significantly the size of our files and also we can scale up and start reusing styles on different files